core/src/main/java/org/apache/calcite/rel/metadata/RelMetadataQuery.java

```diff
@@ -338,25 +338,24 @@ public Double getPercentageOriginalRows(RelNode rel) {
   }
 
   /**
-   * Determines the origin of a column, provided the column maps to a single
-   * column that isn't derived.
+   * Determines the origin of a column, provided the column maps to a origin
+   * column.
    *
    * @see #getColumnOrigins(org.apache.calcite.rel.RelNode, int)
    *
    * @param rel the RelNode of the column
    * @param column the offset of the column whose origin we are trying to
    * determine
    *
-   * @return the origin of a column provided it's a simple column; otherwise,
-   * returns null
+   * @return the origin of a column provided it's a simple column;
```

Comment:
The doc here does not match with the code.

Reply:
Thanks for review, update the doc.